Lib/turtle.py

```diff
@@ -1939,31 +1939,40 @@ def circle(self, radius, extent = None, steps = None):
         """ Draw a circle with given radius.
 
         Arguments:
-        radius -- a number
-        extent (optional) -- a number
+        radius -- a number (distance)
+        extent (optional) -- a number (angle, in degrees)
         steps (optional) -- an integer
+        
+        With one argument, draw a circle with the given radius. The
+        center is radius units to the left of the turtle. If radius is
+        negative, the center is to the right.
+
+        If extent is given, do not draw the whole circle, but only an
+        arc of the circle extent degrees wide starting from the current
+        position. If extent is negative, draw the arc while moving
+        backwards around the circle from the current position. In either
```

Comment:
I think here it would help talking about clockwise and counter-clockwise instead of using "backwards".

Reply:
The trick is that either radius or extent, or both, may be negative.  Any time either is switched from negative to positive or back, the resulting direction of drawing (clockwise or counter-clockwise) will be switched also.